### PR TITLE
[RFR] py3prt

### PR DESCRIPTION
**Pull request by izapolsk**
Shriver: This is built off of #9053

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit b1721e0e28b675790a1dcb39afd7e396feb406d1:

`cfme/fixtures/artifactor_plugin.py`:
- :heavy_exclamation_mark: Line 66:47: **W504** *line break after binary operator*
- :heavy_exclamation_mark: Line 67:40: **W504** *line break after binary operator*

`cfme/markers/meta.py`:
- :heavy_exclamation_mark: Line 137:79: **W504** *line break after binary operator*

`sprout/appliances/tasks.py`:
- :heavy_exclamation_mark: Line 1605:36: **W291** *trailing whitespace*
- :heavy_exclamation_mark: Line 1606:64: **W291** *trailing whitespace*

Hmmm
:scream: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,91 @@
+# https://docs.travis-ci.com/user/languages/python/#python-37-and-higher
```
Comment (kedark3) [inline .travis.yml]: Booo :-1:  no RHEL/CentOS/Fedora. :rofl:

**Comment by dajoRH** [comment]
Lint report for commit e926d7ac4314a8f55b5dc7e6e1a2a698907c68fc:

`sprout/appliances/tasks.py`:
- :heavy_exclamation_mark: Line 1605:36: **W291** *trailing whitespace*
- :heavy_exclamation_mark: Line 1606:64: **W291** *trailing whitespace*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Pull request merged**